src/components/Reports/MPGAIncomeExpensesReport/Charts/SummaryBarChart.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Bar,
+  BarChart,
+  Cell,
+  LabelList,
+  ResponsiveContainer,
+  XAxis,
+  YAxis,
+} from 'recharts';
+
+interface SummaryBarChartProps {
+  incomeTotal: number | undefined;
+  expensesTotal: number | undefined;
+  aspect: number;
+  width: number;
+}
+
+const chartColors = ['#05699B', '#00C0D8'];
+
+export const SummaryBarChart: React.FC<SummaryBarChartProps> = ({
+  incomeTotal,
+  expensesTotal,
+  aspect,
+  width,
+}) => {
+  const { t } = useTranslation();
+
+  const data = [
+    { name: t('Income'), total: incomeTotal },
+    {
+      name: t('Expenses'),
+      total: expensesTotal,
+    },
+  ];
+
+  const [hasMounted, setHasMounted] = useState(false);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+
+  if (!hasMounted) {
+    return null;
+  }
+
+  return (
+    <ResponsiveContainer width={`${width}%`} aspect={aspect}>
+      <BarChart
```

Comment:
[Suggested tests] Test names: 'should format currency according to user locale', 'should handle different currencies (EUR, GBP)', 'should use fallback currency when none provided'. Assertions: Check that `currencyFormat` is called with correct parameters, verify different locale formatting (en-US vs fr-FR), test currency symbol display.